petab/core.py

```diff
@@ -561,10 +559,10 @@ def split_parameter_replacement_list(list_string):
 
     def to_float_if_float(x):
         try:
-            return float(x)
+            ret = float(x)
         except ValueError:
-            return x
-
+            ret = x
+        return ret
```

Comment:
What was the problem? Was cleaner before...

Reply:
oh yes, I had changed that since I assumed it might cause problems. will undo.